todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/local/ToDoDatabase.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017, The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.android.architecture.blueprints.todoapp.data.source.local;
+
+import android.arch.persistence.room.Database;
+import android.arch.persistence.room.Room;
+import android.arch.persistence.room.RoomDatabase;
+import android.content.Context;
+
+import com.example.android.architecture.blueprints.todoapp.tasks.domain.model.Task;
+
+
+/**
+ * The Room Database that contains the Task table.
+ */
+@Database(entities = {Task.class}, version = 1)
+public abstract class ToDoDatabase extends RoomDatabase {
+
+    private static ToDoDatabase INSTANCE;
```

Comment:
I wonder why this variable name is uppercase when it is not a final, just static one?

Reply:
You are right but we need to do that in todo-mvp